detekt-rules/baseline.xml

```diff
@@ -68,8 +68,8 @@
     <ID>AbsentOrWrongFileLicense:ViewObjectVersionAction.kt$software.aws.toolkits.jetbrains.services.s3.objectActions.ViewObjectVersionAction.kt</ID>
     <ID>AbsentOrWrongFileLicense:build.gradle.kts$.build.gradle.kts</ID>
     <ID>AbsentOrWrongFileLicense:settings.gradle.kts$.settings.gradle.kts</ID>
-    <ID>BannedPattern:BannedPatternRule.kt$software.aws.toolkits.gradle.detekt.rules.BannedPatternRule.kt</ID>
-    <ID>BannedPattern:BannedPatternRuleTest.kt$.BannedPatternRuleTest.kt</ID>
+    <ID>BannedPattern:BannedPatternRule.kt$BannedPatternRule.kt</ID>
+    <ID>BannedPattern:software.aws.toolkits.gradle.detekt.rules.BannedPatternRuleTest.kt$.software.aws.toolkits.gradle.detekt.rules.BannedPatternRuleTest.kt</ID>
```

Comment:
why is this fully qualified now?

Reply:
The test and one other actually never had a package declaration on them.

Reply:
https://github.com/aws/aws-toolkit-jetbrains/pull/2829/files#diff-685790fadba62a4eac614164ca1e8b905450e2a8cab3990a6ccdf21312215f71R4-R5